### ISSUE TITLE
Doesn't seem to work out of the box

### ISSUE THREAD

**Issue by Zamiell** [wcandillon/eslint-plugin-ts-exports#13]
Greetings,

I tried giving this plugin a test drive on my existing repository, but it didn't work, even with a red herring that I inserted:

```typescript
export function foo(): void {}
```

For troubleshooting purposes, I tried again on a fresh "Hello World"-style repository, and I couldn't get it to work there either.
I've uploaded this repository here: https://github.com/Zamiell/test
Running `lint.sh` does not yield any errors.

So it looks like this plugin isn't working at all.

Can you confirm?

**Comment by GollyJer** [comment]
I'm in the same boat. Was excited to get this going but it seems it's not working.

**Comment by wcandillon** [comment]
Indeed, the plugin is a bit broken at the moment, the main issue is that I'm not sure how to "cache" the dependency tree. If we do it for every file change, it would take too much resources.

The issue you seem to be experiencing is the opposite, the dependency tree.

**Comment by Zamiell** [comment]
Any updates on this? I've resorted to run ts-prune independently, but it would be very useful to be able to integrate all-the-tools inside of eslint.

**Comment by wcandillon** [comment]
Unfortunately, we don't have a good solution on how to run this within eslint.

**Comment by xanderfehsenfeld** [comment]
> Indeed, the plugin is a bit broken at the moment, the main issue is that I'm not sure how to "cache" the dependency tree. If we do it for every file change, it would take too much resources.

Although it is too resource-intensive, do you mind fixing the yarn dependency so others such as myself can clone, build, and run the repo?

**Comment by Zamiell** [comment]
@xanderfehsenfeld Don't use this repo and use ts-prune instead, it is great and I would recommend it.

**Comment by wcandillon** [comment]
indeed this was a failed experiment as it's not really possible to integrate ts-prune with eslint
